content/docs/setup/kubernetes/upgrade/notice/index.md

```diff
@@ -33,4 +33,6 @@ For an overview of new features introduced with Istio 1.1, please refer to the [
 
 ## Security
 
-- RBAC configuration has been modified to implement cluster scoping.  The `RbacConfig` resource has been replaced with the `ClusterRbacConfig` resource. Refer to [Migrating `RbacConfig` to `ClusterRbacConfig`](/docs/setup/kubernetes/upgrade/steps/#migrating-from-rbacconfig-to-clusterrbacconfig) for migration instructions.
+- In Istio 1.1, we replaced the `RbacConfig` resource with the `ClusterRbacConfig` resource.
```

Comment:
@linsun suggested removing this here: https://github.com/istio/istio.io/pull/4471#discussion_r295069658 - so I removed it.

I think the same should be done here.

What we probably need is a different page (in the future) for deprecations and their impact (which is not an upgrade notice per se).

Cheers
-steve

Reply:
I updated it based on @geeknoid's comment: https://github.com/istio/istio.io/pull/4455#issuecomment-502851039

I think keeping it here help to encourage user to migrate soon and as we mentioned in the doc this step is optional if the customer don't want to do so. But it's important to let customers know that the old API will be deleted in 1.3, so the early they migrate, the less risk they will have.

@liminw @rcaballeromx Do we still want to keep it here?

Reply:
Let's kill it for now, we need to get this in. It's reasonable to remove it since 1.0 is no longer officially supported. In the longer term, we need a policy for this kind of stuff.

Reply:
Thanks!